next-app/src/constants/colorMode.ts

```diff
@@ -0,0 +1 @@
+export const COLOR_MODE = 'colorMode'
```

Comment:
```suggestion
export const COLOR_MODE = 'colorMode' as const
```
요것도 특수한 매직 리터럴이니까 리터럴 타입으로 좁혀서?